reqs/sharing-aspect.toml

```diff
@@ -0,0 +1,18 @@
+[REQ-sharing-geolocation]
+text = '''
+Users should be able to find medias to lend near their location.
+
+When searching for medias, their location will only be given at the city level
+and not more precisely to keep the privacy of users.
+'''
+
+[REQ-sharing-exchange]
+text = '''
+Users should exchange physically the medias.
+
+It implies users should agree to a location where to do the trade.
+
+Both users can then proceed the exchange by knowing the state of each media.
+
+Currently there is not guarantee about a good experience when lending medias.
```

Comment:
What do you mean with this sentence? Do you mean that sharing the media is based on trust?